### ISSUE TITLE
Support for function code via ZipFile in Lambda functions with any dotnet3.1 runtime

### ISSUE THREAD

**Issue by ragasivasankar** [aws-cloudformation/cloudformation-coverage-roadmap#733]
Looking for Support on PowerShell  in function ode ZipFile which help to build Cloud formation more effective automation not depend on the python 3rd party library /license impact .

**Comment by PatMyron** [comment]
Agreed: https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/80#issuecomment-604138808

**Comment by brianterry** [comment]
@ragasivasankar, `dotnetcore3.1`  is now supported.

Please see our docs:
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html

**Comment by PatMyron** [comment]
@brianterry [doesn't look like these runtimes are supported in `ZipFile` yet](https://code.amazon.com/packages/AWS21Resources/blobs/mainline/--/src/com/amazon/aws21/resources/lambda/function/FunctionCustomResource.java#L30)

**Comment by brianterry** [comment]
@PatMyron, reopening the issue and investigating
